test/integration/metrics_integration_test.go

```diff
@@ -0,0 +1,45 @@
+//go:build integration
+// +build integration
+
+package integration
+
+import (
+	"context"
+	"github.com/cloudscale-ch/cloudscale-go-sdk"
+	"reflect"
+	"testing"
+	"time"
+)
+
+func TestIntegrationMetrics_GetBuckets(t *testing.T) {
+	integrationTest(t)
+
+	request := &cloudscale.BucketMetricsRequest{
+		Start:          time.Date(2020, 1, 1, 0, 0, 0, 0, time.UTC),
+		End:            time.Date(2020, 1, 2, 0, 0, 0, 0, time.UTC),
+		BucketNames:    []string{},
+		ObjectsUserIDs: []string{},
+	}
+
+	response, err := client.Metrics.GetBuckets(context.Background(), request)
+	if err != nil {
+		t.Fatalf("Metrics.GetBuckets returned error %s\n", err)
+	}
+
+	expectedResponse := cloudscale.BucketMetrics{
+		Start: time.Date(2019, 12, 31, 23, 0, 0, 0, time.UTC),
+		End:   time.Date(2020, 1, 1, 23, 0, 0, 0, time.UTC),
+		Data: []cloudscale.BucketMetricsData{
+			{
+				Subject: cloudscale.BucketMetricsDataSubject{
+					BucketName:    "Carmichael",
+					ObjectsUserID: "2b44ef90ff15f010dc013f8d32612e07d7734e5f5581d0c94be56780b0dcd58a",
+				},
+			},
+		},
+	}
+
+	if reflect.DeepEqual(response, expectedResponse) {
```

Comment:
caution: I think you are missing a not here.

Reply:
Yes. Fixed.